doc/man3/flux_watcher_start.rst

```diff
@@ -36,6 +42,16 @@ callback.
 :func:`flux_watcher_is_active` returns a true value if the watcher is active
 (i.e. it has been started and not yet stopped) and false otherwise.
 
+:func:`flux_watcher_unref` drops the watcher's reference on the reactor.
+This function is idempotent.  :func:`flux_reactor_run` normally runs until
+there are no more active watchers with references.
+
+:func:`flux_watcher_ref` restores the watcher's reference on the reactor.
+This function is idempotent.
+
+:func:`flux_watcher_is_referenced` returns true if the watcher is referenced.
+All watchers are referenced unless updated with :func:`flux_watcher_unref`.
+
```

Comment:
Add note that not all watchers may support `flux_watcher_unref()`? Perhaps check `flux_watcher_is_referenced()` to ensure the function has had the desired effect. (or do all watchers in practice now support this call?)

Reply:
All watchers in the public API support this call, but I didn't bother with the private ones (fbuf, sdbus).  I'll make a note to that effect.